apps/web/src/bridge/listeners/index.ts

```diff
@@ -93,6 +93,8 @@ export const getMessageOrTransaction = async (requestId: string) => {
 			};
 		} else if ('transaction' in res) {
 			return res;
+		} else {
+			return res;
 		}
```

Comment:
both of these else statments return `res`

Reply:
the former will be updated with transaction decode feature, this will take more time to research